src/components/TopMenu/TopMenu.js

```diff
@@ -1,25 +1,67 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { Menu } from "antd";
 import { Link } from "react-router-dom";
 import styled from "styled-components";
+import { withRouter } from "react-router";
+const employ_logo =
+  process.env.PUBLIC_URL + "images/temporaryImages/employ_logo.svg";
+const lines_layout =
+  process.env.PUBLIC_URL + "images/temporaryImages/lines_layout.png";
 
 const StyledMenu = styled(Menu)`
   line-height: 45px;
+  ${({ show }) => show === "false" && `visibility: hidden;`}
 `;
-
-function TopMenu() {
+const HeaderLayer = styled.div`
+  display: none;
```

Comment:
```suggestion
  display: ${props => props.show === "true" ? "none" : "block"}
```

Reply:
After the changes, I have an error 
![image](https://user-images.githubusercontent.com/19416734/72365018-23e59e80-3700-11ea-8241-19013a08bff6.png)